src/app/components/Curation/index.test.tsx

```diff
@@ -165,6 +179,67 @@ describe('Curation', () => {
     });
   });
 
+  describe('Billboard', () => {
+    it('should not be displayed if there are no promos', () => {
+      render(
+        <Curation
+          position={0}
+          visualStyle={BANNER}
+          visualProminence={MAXIMUM}
+          summaries={[]}
+        />,
+      );
+
+      expect(
+        document.querySelector('[data-testid="billboard-"]'),
+      ).not.toBeInTheDocument();
+    });
+
+    it('should not be displayed if the summaries returned are empty', () => {
+      render(
+        <Curation
+          position={0}
+          visualStyle={BANNER}
+          visualProminence={MAXIMUM}
+          // @ts-expect-error this test checks for empty summaries hence there being no fields passed into the summary object
+          summaries={[{}]}
+        />,
+      );
+
+      expect(
+        document.querySelector('[data-testid="billboard-"]'),
+      ).not.toBeInTheDocument();
+    });
```

Comment:
This scenario will never occur, as we will never have a summary array with no content - either the array is empty, or is has data. Let's delete to avoid confusion.

```suggestion
```